AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DisplayMapperAdded.py

```diff
@@ -98,20 +157,202 @@ def AtomEditorComponentsLevel_DisplayMapper_AddedToEntity():
             display_mapper_component.get_component_property_value(
                 AtomComponentProperties.display_mapper('LDR color Grading LUT')) == display_mapper_asset.id)
 
-        # 5. Set Enable LDR color grading LUT property True
+        for operation_type in DISPLAY_MAPPER_OPERATION_TYPE.keys():
+            # 5. Set the Display Mapper Operation Type enumerated in DISPLAY_MAPPER_OPERATION_TYPE for each type
+            # Type property cannot be set currently. as a workaround we are calling an ebus to set the operation type
```

Comment:
Will this be fixed/changed in the future? We should leave a comment indicating such (without internal info).

Reply:
There is in progress work on this. I'll have a GHI for it.